shared/ui/combobox/combobox.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { ChangeEvent, FC, MouseEventHandler, useState } from 'react';
+import { clsx } from 'clsx';
+
+import { Combobox as ComboboxUI } from '@headlessui/react';
+import * as ScrollArea from '@radix-ui/react-scroll-area';
+
+import style from './combobox.module.scss';
+
+import { Typography } from '..';
+
+import ArrowDown from '../../../assets/icons/arrow-down.svg?url';
```

Comment:
Prefer using SVG-based JSX-elements if it is possible. [Mentilned here](https://github.com/Iplatos/inctagram/pull/79#discussion_r1451766682).

Reply:
Maybe we should change assets folder with all icons to JSX-elements?

Reply:
Don't rush to redo everything just yet. There are also large images among the assets, which color manipulation is not necessary. There is no point in wrapping them in JSX. Instead, it's better to insert them using the `src` prop of the `NextImage` component, as it allows you to optimize the size of static assets and also caches them.

Besides, it's a big piece of work to wrap all the icons in JSX. Especially since some of them are crookedly downloaded. It's better to do it gradually, keeping the original svg file. If in time we fix the "svgr" plugin for the storybook, we will delete all .tsx files of icons.